nats-top.yaml

```diff
@@ -0,0 +1,91 @@
+package:
+  name: nats-top
+  version: "0.6.3"
+  epoch: 0
+  description: A top-like tool for monitoring NATS servers.
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/nats-io/nats-top
+      tag: v${{package.version}}
+      expected-commit: 812d35cc991232d7606816a15b2ea0b12fb129e7
+
+  - uses: go/build
+    with:
+      packages: .
+      output: nats-top
+      ldflags: -X main.version=${{package.version}} -X main.commit=$(git rev-parse HEAD) -X main.date=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compat package for ${{package.name}}
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/local/bin
+          ln -s /usr/bin/nats-top "${{targets.contextdir}}"/usr/local/bin/nats-top
+
+update:
+  enabled: true
+  github:
+    identifier: nats-io/nats-top
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - nats-server
+        - nats
+        - util-linux-misc
+        - mkcert
+        - openssl
+  pipeline:
+    - name: "Version test"
+      runs: |
+        nats-top --version 2>&1 | grep ${{package.version}}
+    - name: Test nats-top with minimal nats server and 2 clients
+      runs: |
+        nats-server -p 4222 -m 8222 & sleep 2 &
+        nats sub client1 > /dev/null & sleep 2 & nats sub client2 > /dev/null & sleep 2
+        nats-top -s localhost -b -u -o nats-output-${{package.version}}.txt
+        cat nats-output-${{package.version}}.txt
+    - name: test connection limit (-n) and sorting options (-sort)
+      runs: |
+        nats-top -n 1 -sort uptime -o nats-output-${{package.version}}.txt
```

Comment:
I would test those options separately as with one connection the sorting result wouldn't change.

Something like

```yml
    - name: test connection limit (-n)
      runs: |
        nats-top -n 1 -o output.txt
        grep -q "Connections Polled: 1" output.txt
    - name: test sorting options (-sort)
      runs: |
        nats-top -sort uptime -u -o output.txt
        grep -q client2 <(tail -1 output.txt)
```